modules/data-entry/src/main/frontend/src/questionnaire/Subject.jsx

```diff
@@ -241,14 +244,14 @@ function SubjectMember (props) {
   if (level == 1) {buttonSize = "medium"; headerStyle="h4"};
   if (level > 1) {buttonSize = "small"; headerStyle="h5"};
 
-  let parentDetails = data && data['parents'] && getHierarchy(data['parents'], Link, (node) => ({to: "/content.html" + node["@path"]}));
+  let parentDetails = data && data['parents'] && getHierarchy(data['parents'], Link, (node) => ({href: "/content.html" + node["@path"], target :"_blank"}));
 
   return (
     <Grid item xs={12}>
       <Grid item>
         <span className={classes.subjectHeader}>
           {
-            parentDetails && <Typography variant="overline">{parentDetails}</Typography>
+            (originType != "Patient") && parentDetails && <Typography variant="overline">{parentDetails}</Typography>
```

Comment:
Per previous comment, wouldn't this work instead? (didn't test it)
```suggestion
            (level == 0) && parentDetails && <Typography variant="overline">{parentDetails}</Typography>
```